app/admin.js

```diff
@@ -208,7 +208,6 @@ const saveKeys = co(function *(keys, type) {
       type: type,
       pub: key.pub,
       path: key.path,
-      keyid: key.keyid,
```

Comment:
Why don't we need keyid anymore?

Reply:
its a redundant field, was conveying the same information that "path" does

Reply:
regarding optional email suppression - this was a necessary feature for our implementation because my mailgun account couldn't send out emails to non-whitelisted addresses, so i disabled emails altogether. **this is the branch that is live right now**, so i figured it should be included in this PR because it was easier to keep track of which branch is live on google cloud instead of having more unmerged branches floating around and messing something up next time we update the docker image.

Reply:
ideally this branch would have been merged before going live, but it didn't get merged in time. and i wasn't going to delay the launch further for something simple like that.